server/src/config-helper.ts

```diff
@@ -111,7 +111,7 @@ const BASE_CONFIG: CommonVoiceConfig = {
     castBoolean
   ),
   TRANSCODE: {
-     AUDIO_CODEC: configEntry('CV_TRANSCODE_CODEC', 'mp3'),
+     AUDIO_CODEC: configEntry('CV_TRANSCODE_AUDIO_CODEC', 'mp3'),
```

Comment:
I got worried about `mp3` and `44100` but I see they're just the default values. We could probably actually change the defaults here too

Reply:
I would prefer to make the lease amount of changes to the code.  Changing the audio codec is already hardcoded in our github workflow.